app/controllers/close_contacts_controller.rb

```diff
@@ -3,17 +3,16 @@
 # CloseContactsController: close contacts
 class CloseContactsController < ApplicationController
   before_action :authenticate_user!
+  before_action :check_can_create, only: %i[create destroy]
```

Comment:
I would add a `:check_can_create` for update and destroy that uses `can_edit_patient_close_contacts?`